titus-server-master/src/main/java/com/netflix/titus/master/service/management/internal/ResourceConsumptionEvaluator.java

```diff
@@ -115,8 +115,7 @@ private CompositeResourceConsumption buildSystemConsumption(Map<String, Map<Stri
         capacityGroupConsumptionMap.forEach((capacityGroup, appConsumptions) -> {
 
             ApplicationSLA sla = applicationSlaMap.get(capacityGroup);
-            double buffer = getBuffer(sla.getTier());
```

Comment:
`getBuffer()` can probably be deleted now

Reply:
+1, not going to change it now as this component will be getting an overhaul soon